qt/scientific_interfaces/Indirect/IndirectPlotOptionsView.cpp

```diff
@@ -68,6 +68,9 @@ void IndirectPlotOptionsView::setupView() {
   connect(m_plotOptions->cbWorkspace, SIGNAL(currentIndexChanged(QString const &)), this,
           SLOT(emitSelectedWorkspaceChanged(QString const &)));
 
+  connect(m_plotOptions->cbPlotUnit, SIGNAL(currentIndexChanged(QString const &)), this,
```

Comment:
It looks like this function has been made obsolete: https://doc.qt.io/qt-5/qcombobox-obsolete.html

Can we change to use this in this file? https://doc.qt.io/qt-5.15/qcombobox.html#currentIndexChanged

Reply:
> Looks good, just one comment regarding an obsolete `qt` function which you may want to switch out if it's not too much effort.
> 
> In addition, I suggest raising an issue regarding the error I encountered having an incorrect instrument - the error should handled and stated explicitly given it's so easy to encounter.

The error happens in an algorithm so I think printing the error in the log makes sense. What is the better way to state the error? Is it by showing a pop-up menu?

Reply:
If it isn't easy to catch the exception and then print a clearer message to the log, perhaps we could add validation with regards to the instrument before the algorithm is executed.

The error message currently output to log is cryptic, it isn't immediately clear that a user could do something so simple as changing the instrument to correct it.

If this error existed before this PR I'd raise another issue for this.

Reply:
ok, thanks. I've raised a new issue here #35480